debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/ConnectionContext.java

```diff
@@ -132,8 +143,11 @@ public String hosts() {
         return config.getString(MongoDbConnectorConfig.HOSTS);
     }
 
-    public int pollPeriodInSeconds() {
-        return config.getInteger(MongoDbConnectorConfig.POLL_INTERVAL_SEC);
+    public long pollPeriodInMilliSeconds() {
```

Comment:
Let's convert it to `Duration` right here. In general the approach should be to represent temporal data using the strong semantical types like `Duration` internally, and only convert from/to ms and other representations at the system boundaries.